src/NuGet.Core/NuGet.Commands/TrustedSignersCommand/TrustedSignersCommandRunner.cs

```diff
@@ -10,9 +10,11 @@
 using System.Threading.Tasks;
 using NuGet.Common;
 using NuGet.Configuration;
+#if IS_SIGNING_SUPPORTED
 using NuGet.Packaging;
-using NuGet.Packaging.Signing;
 using NuGet.Protocol;
+#endif
+using NuGet.Packaging.Signing;
```

Comment:
Personally, I'd rather see the conditional using statements to be moved after the unconditional statements, so that they're all one continuous block. This change breaks the "keep using alphabetical" rule.

Also, I'm curious if someone modifies the file and needs to add a new using, will the intellicode auto fix add the new using statement alphabetically or not? 

My comment applies to everywhere that conditional using statments are added, but this is the first one I've seen that makes the usings non-alphabetical.